### PR TITLE
Configuration files for tools to submit in Galaxy ToolShed

### PR DESCRIPTION
**Pull request by anilthanki**

### CODE REVIEW

File: tools/TreeBest/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+categories:
+- Phylogenetics
+description: treebest_best_wrapper
+long_description: |
+    Generate a phylogenetic tree using CDS alignment and species tree.
+name: treebest_best_wrapper 
+owner: Earlham Institute 
+remote_repository_url: https://github.com/TGAC/tgac-galaxytools/tree/master/tools/TreeBest
+typea unrestricted
```
Comment (nsoranzo) [inline tools/TreeBest/.shed.yml]: s/typea/type:/

File: tools/Ensembl-REST/.shed.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Ensembl-REST
-owner: tgac
+owner: Earlham Institute
```
Comment (nsoranzo) [inline tools/Ensembl-REST/.shed.yml]: Mmmh, `tgac` is the ToolShed user "public name", which cannot be changed unfortunately. Should I create a new ToolShed user? `tgac` has 2 published repositories (hcluster_sg and miranda), we can deprecate them and publish them again under the new user.

File: tools/ete/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+categories:
+- Phylogenetics
+description: Generates species tree from a list of species using the ETE Toolkit
+long_description: |
+	Generate a species tree from a list of species using the `ETE Toolkit`
```
Comment (nsoranzo) [inline tools/ete/.shed.yml]: Please change tabs to spaces (also in following files). Tabs are forbidden in YAML.

File: tools/Ensembl-REST/.shed.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+name: Ensembl-REST
+owner: Earlham Institute
+description: Ensembl-REST wrappers
+long_description: |
+    http://rest.ensembl.org
+remote_repository_url: https://github.com/TGAC/tgac-galaxytools/tree/master/tools/Ensembl-REST
```
Comment (nsoranzo) [inline tools/Ensembl-REST/.shed.yml]: `s|tgac-galaxytools|earlham-galaxytools|` on all files

File: tools/Ensembl-REST/.shed.yml (hunk 2 above)
Comment (anilthanki) [inline tools/Ensembl-REST/.shed.yml]: ohhh... I dont know then...up-to you

File: tools/Ensembl-REST/.shed.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+name: Ensembl-REST
+owner: earlhaminst
+description: Ensembl-REST wrappers
+long_description: http://rest.ensembl.org
+remote_repository_url: https://github.com/TGAC/earlham-galaxytools/tree/master/tools/Ensembl-REST
+type: unrestricted
+auto_tool_repositories:
+  name_template: "Ensembl {{ tool_id }}"
```
Comment (nsoranzo) [inline tools/Ensembl-REST/.shed.yml]: Remove `"Ensembl `

Comment (nsoranzo) [inline tools/Ensembl-REST/.shed.yml]: `s/Ensembl /ensembl_/`

File: tools/Ensembl-REST/.shed.yml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+  description_template: "Wrapper for Ensebl-REST {{ tool_name }}."
```
Comment (nsoranzo) [inline tools/Ensembl-REST/.shed.yml]: What about: `"{{ tool_name }} (from the Ensebl-REST tool suite)"` which would result in:
- "Get Ensembl features by ID (from the Ensebl-REST tool suite)"
- "Get gene tree by Ensembl ID (from the Ensebl-REST tool suite)"
- "Get sequences by Ensembl ID (from the Ensebl-REST tool suite)"

File: tools/Ensembl-REST/.shed.yml [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (nsoranzo) [inline tools/Ensembl-REST/.shed.yml]: Add `categories:` field

File: tools/hcluster_sg/.shed.yml [hunk 8]
```diff
@@ -1,11 +1,8 @@
 categories:
 - Phylogenetics
 description: Hierarchically clustering on a sparse graph
-long_description: |
-    hcluster_sg is a tool for hierarchically clustering on a sparse graph.
-    
-    https://github.com/douglasgscofield/hcluster
+long_description: hcluster_sg is a tool for hierarchically clustering on a sparse graph. https://github.com/douglasgscofield/hcluster
```
Comment (nsoranzo) [inline tools/hcluster_sg/.shed.yml]: This change is not needed, only tabs are a problem, indentation with (2 or 4) spaces is fine.

**Pull request merged**